plugins/woocommerce/client/blocks/bin/copy-blocks-json.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# This script mimics the CopyWebpackPlugin behavior from the WooCommerce Blocks webpack configuration
+# to make block.json files available for unit testing without requiring a full build.
+# Ensure that the logic of this script is kept in sync with the logic of the CopyWebpackPlugin in the WooCommerce Blocks webpack configuration:
+# https://github.com/woocommerce/woocommerce/blob/84d1da7be3cbd3d8f40b17ad58729f668fd82b6a/plugins/woocommerce/client/blocks/bin/webpack-configs.js#L229-L256
+
+# Move to the project root
+while [ ! -d "plugins/woocommerce" ] || [ ! -f "pnpm-workspace.yaml" ]; do
+    if [ "$PWD" = "/" ]; then
+        echo "Error: Could not find project root"
+        exit 1
+    fi
+    cd ..
+done
+
+# Set target directory
+TARGET_DIR="plugins/woocommerce/assets/client/blocks"
+
+# Create target directory if it doesn't exist
+mkdir -p "$TARGET_DIR"
+
+# Find all block.json files
+find plugins/woocommerce/client/blocks/assets/js -name "block.json" | while read file; do
+    # Read the block name from the JSON file
+    block_name=$(cat "$file" | grep -o '"name": "[^"]*"' | cut -d'"' -f4 | cut -d'/' -f2)
+
+    # Check if it's a parent block by looking for "parent" field
+    if grep -q '"parent":' "$file"; then
```

Comment:
If it needs to mimic the CopyWebpackPlugin behavior, I think we should also include the second condition there which is `! genericBlocks[ blockName ]`. Otherwise these generic blocks won't have `block.json` in the right spot and problem won't be addressed for them, no? 🤔 

```
if (
	metadata.parent &&
	! genericBlocks[ blockName ]
)
```

`genericBlocks` are [listed here](https://github.com/woocommerce/woocommerce/blob/2afc098512d2f6782339ca0b34c9320c70acdb57/plugins/woocommerce/client/blocks/bin/webpack-entries.js#L212).

Reply:
Nice catch! I fixed with 3c1fe64fbdac3ded43fded07f6b610a749bebae2